.github/workflows/build-cli-dotnet.yaml

```diff
@@ -31,12 +31,4 @@ jobs:
       run: dotnet build --no-restore
 
     - name: Unit Tests
-      run: dotnet test --no-build --verbosity normal
-      
-    - name: Regression Tests 'Get Emission Data for AZure Region EastUs'
-      run: ./bin/Debug/net6.0/CarbonAwareCLI -l eastus -d ./bin/Debug/net6.0/data-files/dummy-data-azure-emissions.json
-      working-directory: './src/CarbonAware.CLI/'
-
-    - name: Regression Tests 'Get lowest Emission for several Regions eastus,francecentral,brazil,ukwest'
-      run: ./bin/Debug/net6.0/CarbonAwareCLI -l eastus,francecentral,brazil,ukwest  -d ./bin/Debug/net6.0/data-files/dummy-data-azure-emissions.json --lowest
-      working-directory: './src/CarbonAware.CLI/'
\ No newline at end of file
+      run: dotnet test --no-build --verbosity normal
```

Comment:
was mainly curious to see, that this agrees with the "Auto-PR" having detected this [CLI GitHub Action](https://github.com/microsoft/carbon-aware-sdk/pull/118/files) had changed :)